src/main/java/namor/template/calculator/Sign.java

```
@@ -0,0 +1,28 @@
+package namor.template.calculator;
+
+public enum Sign {
+    ADDITION("+"),
+    SUBTRACTION("-"),
+    MULTIPLICATION("*"),
+    DIVISION("/"),
+    NUMBER(" ");
+
+    String key;
+
+    Sign(String key){
+        this.key = key;
+    }
+
+    public String getKey() {
+        return key;
+    }
+
+    public static Sign getValue(String key) {
+        for(Sign sign : Sign.values()){
+            if (sign.getKey().equals(key)){
+                return sign;
+            }
+        }
+        return NUMBER;
```

Should I make method that cheks if my "key" is number?

---

You can throw an exception if there is nothing to return.